CNS-784: Projects README (#1070)
lavanet/lava: x/plans/README.md

The `Type` field lets the user define APIs that have different functionalities depending on their type. the valid types are: `GET` and `POST`. An example of such API is Cosmos' `/cosmos/tx/v1beta1/txs` API. If it's sent as a `GET` request, it fetches transactions by event and if it's sent as a `POST` request, it sends a transaction.

> The `AddOn` field lets you use additional optional APIs (debug, trace and more). Overall, the add-ons can be: `debug`, `trace` and `convex`.

Reviewer: ```suggestion
The `AddOn` field lets you use additional optional APIs, which can be: `debug`, `trace` and `convex`.
```

Author: done. see https://github.com/lavanet/lava/commit/8716c13495e79fb3c836254917cb6de2553db485
